portfolio.js

```diff
@@ -0,0 +1,26 @@
+const main = document.querySelector('main');
+const hamburger = document.querySelector('#hamburger');
+const close = document.querySelector('#close');
+const portfolio = document.querySelector('#portfolio');
+const about = document.querySelector('#about');
+const contact = document.querySelector('#contact');
+
+hamburger.addEventListener('click', () => {
+  main.classList.add('active');
+});
+
+close.addEventListener('click', () => {
+  main.classList.remove('active');
+});
+
+portfolio.addEventListener('click', () => {
+  main.classList.toggle('active');
+});
+
+about.addEventListener('click', () => {
+  main.classList.toggle('active');
+});
+
+contact.addEventListener('click', () => {
+  main.classList.toggle('active');
+});
```

Comment:
**[Optional]**

**You are applying the same event listener for 5 times. This violates the principle of DRY.**

What you can do is, you can store all the elements in an array and then loop through them to apply the same event listener. You can also use event delegation.